catalogs/catalog.yml

```diff
@@ -68,17 +83,43 @@ sources:
       urlpath: s3://city-planning-entitlements/gis/source/la_intersections.geojson
   toc_tiers:
     driver: shapefile
-    description: TOC Tiers from October 2017
+    description: TOC Tiers from October 2017. Refer to data-workflow-notes for creating TOC tiers file.
     args:
       urlpath: zip+s3://city-planning-entitlements/gis/source/TOC_Tiers_Oct2017.zip
+  # TOC Analysis
+  toc_parcels_with_entitlements:
+    driver: geojson
+    description: TOC-eligible parcels with entitlement data attached.
+    args:
+      urlpath: s3://city-planning-entitlements/gis/intermediate/toc_eligible_parcels_with_entitlements.geojson
+  # Census
+  # NEED TO FIX THIS BECAUSE IT'S GEOPARQUET, needs to be downloaded first
+  census_analysis_table:
+    driver: parquet
+    description: Census data reshaped to tract-level using 2018 ACS data, in notebooks/B1-census-tract-stats.ipynb.
+    args:
+      urlpath: s3://city-planning-entitlements/data/final/census_analysis_table.parquet
+      engine: pyarrow
```

Comment:
@ian-r-rose Are we able to upload geoparquet to catalog? If not, I can make this a regular parquet, and we just always merge in the geometry using census_tracts.

Reply:
Using parquet, so we still have to merge in `census_tracts` from the catalog to get the geometry. Would rather have census_analysis_table in the catalog, as opposed to always downloading from S3, then reading it in.

Reply:
I think that you are right that geopandas doesn't support reading parquet directly from s3 yet. I'd have to upstream that fix. For the time being, either workaround should be okay, I think

Reply:
For the time being, I'm putting read-able s3 files in the catalog.